NAMESPACE

```diff
@@ -6,7 +6,11 @@ export(cllr_rename_id)
 export(fetch_ats)
 export(fetch_csv)
 export(fetch_vectronics)
+export(filter_date_range)
+export(filter_last_loc)
 export(morph_gps)
+export(produce_gpx)
+export(produce_map)
```

Comment:
Would "make" be a shorter prefix, but still retain the same meaning?

Reply:
Yes - I'll switch to make.